### PR TITLE
Update vSphere user permissions document with specific granular permi…

### PR DESCRIPTION
**Pull request by ptrivedi**
…ssions required

for EKS-A and role creation/modification instructions

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/reference/vsphere/user-permissions.md [hunk 1]
```diff
@@ -5,49 +5,117 @@ description: >
...
+In order to assign specific permissions to an EKS Anywhere vSphere user account, you will first need to create a role using the vCenter console. Alternatively, you may decide to use an existing role and set the necessary permissions on it.
+
+### Create Role
+
+Go to Administration->Access Control->Roles. Click on '+' to add a new role. Select the desired permissions category from the left panel, and specific permissions within the category using the checkbox items on the right panel.
+
+### Modify Role Permissions
+
+In order to modify permissions on an existing role, 
+Go to Administration->Access Control->Roles. Select the desired role and click on the 'Edit role action' icon that resembles a pencil. Select the desired permissions category from the left panel, and specific permissions within the category using the checkbox items on the right panel.
```
Comment (chrisnegus) [inline docs/content/en/docs/reference/vsphere/user-permissions.md:19]:
```suggestion
go to Administration->Access Control->Roles. Select the desired role and click on the 'Edit role action' icon that resembles a pencil. Select the desired permissions category from the left panel, and specific permissions within the category using the checkbox items on the right panel.
```

File: docs/content/en/docs/reference/vsphere/user-permissions.md [hunk 2]
```diff
@@ -5,49 +5,117 @@ description: >
...
+
+### Assign Role to User/Group
+
+Select Administration->Access Control->Global Permissions. Select the desired User/Group and click on the 'Change Role' icon that resembles a pencil. Select the desired role from the Roles drop-down to assign to the User/Group.
+
+
+## EKS Anywhere vSphere User Permissions
+
+Below are the permissions needed by the role assigned to an EKS Anywhere vSphere user to be able to successfully create/update/delete EKS-A clusters.
```
Comment (chrisnegus) [inline docs/content/en/docs/reference/vsphere/user-permissions.md:28]:
```suggestion
Below are the permissions needed by the role assigned to an EKS Anywhere vSphere user to be able to successfully create/update/delete EKS Anywhere clusters.
```

**Review by chrisnegus — COMMENTED** [review]
I just had a couple of small suggestions. Otherwise /lgtm. I'd like to get a technical reviewer to look at this as well.

**Review by TerryHowe — APPROVED** [review]
/lgtm

**Pull request merged**